src/DynamoCoreWpf/ViewModels/Search/NodeAutoCompleteSearchViewModel.cs

```diff
@@ -85,9 +85,17 @@ internal IEnumerable<NodeSearchElement> GetMatchingSearchElements()
         {
             var elements = new List<NodeSearchElement>();
             var inputPortType = PortViewModel.PortModel.GetInputPortType();
+
+            //List of nodes that are skipped temporarily, and will display list of default suggestions instead.
+            var skippedInputTypes = new List<string>() { "var", "object", "string", "bool", "int", "double" };
+
+            if (inputPortType == null || (skippedInputTypes.Any(s => s == inputPortType.ToString())))
+            {
+                return elements; 
+            }
```

Comment:
I think these lines of code need to be moved down to line 104, i.e. after lines:
```
//if parsing the type failed, revert to original string.
inputPortType = ast != null ? ast.datatype.Name : inputPortType;
```
At this point, the rank is stripped from the type info.

Reply:
@aparajit-pratap Can you explain why you prefer moving the code in details? If we are simply ignoring these types, why run the algorithm against these types at all?